.github/workflows/units.yaml

```diff
@@ -0,0 +1,38 @@
+name: Monocle CI
+on: push
+
+jobs:
+  runner-job:
+    runs-on: ubuntu-latest
```

Comment:
why ubuntu?

Reply:
Because of no choice: https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners#supported-runners-and-hardware-resources